python/plugins/MetaSearch/ui/newconnectiondialog.ui

```diff
@@ -68,7 +68,11 @@
        </widget>
       </item>
       <item row="2" column="1">
-       <widget class="QLineEdit" name="lePassword"/>
+       <widget class="QgsPasswordLineEdit" name="lePassword">
+        <property name="echoMode">
+         <enum>QLineEdit::Password</enum>
+        </property>
+       </widget>
```

Comment:
```suggestion
       <widget class="QgsPasswordLineEdit" name="lePassword"/>
```

According to the discussion on gitter @tomkralidis ?
Otherwise, I was thinking the "eye" graphic icon might not be synchronized with the status of showing or not the password :)

Reply:
Thanks @Gustry makes sense.  Changes pushed.